ResidentsSocialCarePlatformApi.Tests/V1/UseCase/GetAllCaseNotesUseCaseTests.cs

```diff
@@ -0,0 +1,45 @@
+using System.Linq;
+using AutoFixture;
+using FluentAssertions;
+using Moq;
+using ResidentsSocialCarePlatformApi.V1.Boundary.Requests;
+using ResidentsSocialCarePlatformApi.V1.Factories;
+using ResidentsSocialCarePlatformApi.V1.Gateways;
+using ResidentsSocialCarePlatformApi.V1.UseCase;
+using NUnit.Framework;
+using ResidentsSocialCarePlatformApi.V1.Domain;
+using ResidentInformation = ResidentsSocialCarePlatformApi.V1.Domain.ResidentInformation;
+using ResidentInformationResponse = ResidentsSocialCarePlatformApi.V1.Boundary.Responses.ResidentInformation;
+
+namespace ResidentsSocialCarePlatformApi.Tests.V1.UseCase
```

Comment:
What happens if the gateway returns an empty list? I think we're missing a test that covers this case.

Reply:
👍🏾